tests/features/class-wc-tests-ces-tracks.php

```diff
@@ -26,16 +26,15 @@ class WC_Tests_CES_Tracks extends WC_Unit_Test_Case {
 	public function setUp() {
 		parent::setUp();
 		update_option( 'woocommerce_allow_tracking', 'yes' );
-		$this->ces = new CustomerEffortScoreTracks();
```

Comment:
I moved instantiation of `CustomerEffortScoreTracks` to the individual tests because the constructor has side-effects. 

When it was in `setUp`, a `woocommerce_update_options` action was added prior to every test. This interfered with the mobile test, which needs to check that mobile usage has no observable effects.